target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -110,7 +110,7 @@ ipq40xx_setup_interfaces()
 		ucidef_set_interface_lan "sw-eth1 sw-eth2"
 		;;
 	netgear,lbr20)
-		ucidef_set_interface_lan "lan1 lan2"
+		ucidef_set_interfaces_lan_wan "lan" "wan"
 		ucidef_set_interface "wan" device "/dev/cdc-wdm0" protocol "qmi"
```

Comment:
Wrong. WAN is set as cellular modem by default. You have it on the line above, I guess you didn't use lbr20.